tox.ini

```diff
@@ -0,0 +1,12 @@
+[tox]
+envlist = py27, py36, py38
+
+[testenv]
+whitelist_externals = make
+deps = -rrequirements-dev.txt
+commands =
+    make test
+
+[testenv:py27]
+deps = -rrequirements-dev-py27.txt
```

Comment:
I prefer one requirements-dev.txt file when we can use environment markers:

`some-package; python_version < '3'`